components/icons/Add.js

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
```

Comment:
Is the import of the whole React necessary? Where do you use it?

Reply:
this is an automated process from a webpack i use to transform my svgs